builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -49,6 +49,25 @@ func TestAccAWSDBInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstance_withoutEngineVersion(t *testing.T) {
```

Comment:
@catsby with regards to this, how can I check that an attribute is present? I don't know what the value will be - should I just use a regex?